File: core/src/org/rexcellentgames/burningknight/entity/item/accessory/equippable/TimeBullet.java
package org.rexcellentgames.burningknight.entity.item.accessory.equippable;

public class TimeBullet extends Equippable {
	@Override
	public void onEquip(boolean load) {
		super.onEquip(load);
		this.owner.modifyStat("slow_down_on_hit", 1f);
	}

	@Override
	public void onUnequip(boolean load) {
		super.onUnequip(load);
		this.owner.modifyStat("slow_down_on_hit", -1f);
	}
}

File: core/src/org/rexcellentgames/burningknight/entity/item/accessory/equippable/TimeBullet.kt
package org.rexcellentgames.burningknight.entity.item.accessory.equippable

class TimeBullet : Equippable() {
	override fun onEquip(load: Boolean) {
		super.onEquip(load)
		this.owner.modifyStat("slow_down_on_hit", 0.1f)
	}

	override fun onUnequip(load: Boolean) {
		super.onUnequip(load)
		this.owner.modifyStat("slow_down_on_hit", -0.1f)
	}

	override fun canBeUpgraded(): Boolean {
		return false
	}
}